tests/plugins/mathjax/manual/mathjax3.md

```diff
@@ -0,0 +1,13 @@
+@bender-tags: 1441, feature, 4.21.1, mathjax
+@bender-ui: collapsed
+@bender-ckeditor-plugins: widget, dialog, mathjax
```

Comment:
The editor in the test does not work correctly due to the lack of the `wysiwygarea` plugin (which enables actual text editing). Also `widget` and `dialog` plugins are not needed on the list as they should be loaded automatically:

```suggestion
@bender-ckeditor-plugins: mathjax, wysiwygarea
```